packages/jest-emotion/src/utils.js

```diff
@@ -189,15 +203,17 @@ export function getStylesFromClassNames(
 }
 
 export function getStyleElements(): Array<HTMLStyleElement> {
-  let elements = Array.from(document.querySelectorAll('style[data-emotion]'))
+  const elements = isBrowser
```

Comment:
I'm not sure about this, I feel like if we're not in a browser environment when serializing an element we should throw.

Reply:
while i agree with you, i feel we should do it at a higher level if we are to do so, with an informative error message. should i add such to this PR?

Reply:
If by higher level you mean check in the module scope then I don't think that would be ideal because the module that the serializer is in should be runnable outside of a browser environment so that a user can add the serializer to the `snapshotSerializers` option but still have some tests that use the node jest environment. An informative message would be great though.

Reply:
OK, added a warning.